src/rust/engine/process_execution/src/local.rs

```diff
@@ -566,6 +569,8 @@ pub trait CapturedWorkdir {
       }
     }
 
+    let result_metadata = ProcessResultMetadata::new(Some(exec_time_elapsed.into()));
```

Comment:
Shouldn't need the `into()` here: `elapsed` returns a `Duration`.

Reply:
An `std::time::Duration`, we need a `concrete_time::Duration`.